webapp/client/src/pages/PersonBHasDisabilityPage.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import FormFieldRadio from "../components/FormFieldRadio";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import Details from "../components/Details";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function PersonBHasDisabilityPage({
+  form,
+  fields,
+  prevUrl,
+  stepHeader,
+}) {
+  const { t } = useTranslation();
+
+  const tbold = function (key) {
```

Comment:
One more thing that I forgot: This throws a warning because, according to eslint, it isn't the correct way to write a function: `src/pages/PersonAHasDisabilityPage.js  Line 19:17:  Unexpected unnamed function  func-names`